packages/react-native-editor/__device-tests__/gutenberg-editor-file.test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Internal dependencies
+ */
+import EditorPage from './pages/editor-page';
+import { setupDriver, isLocalEnvironment, stopDriver } from './helpers/utils';
+import testData from './helpers/test-data';
+
+jest.setTimeout( 1000000 );
+
+describe( 'Gutenberg Editor File Block tests @canary', () => {
+	let driver;
+	let editorPage;
+	let allPassed = true;
+	const fileBlockName = 'File';
+
+	// Use reporter for setting status for saucelabs Job
+	if ( ! isLocalEnvironment() ) {
+		const reporter = {
+			specDone: async ( result ) => {
+				allPassed = allPassed && result.status !== 'failed';
+			},
+		};
+
+		jasmine.getEnv().addReporter( reporter );
+	}
+
+	beforeAll( async () => {
+		driver = await setupDriver();
+		editorPage = new EditorPage( driver );
+	} );
+
+	it( 'should be able to see visual editor', async () => {
+		await expect( editorPage.getBlockList() ).resolves.toBe( true );
+	} );
+
+	it( 'should be able to add a file block', async () => {
```

Comment:
Hi @etoledom Do we want to split the responsibility of this test into two? i.e one test just adds the File block and checks if the element is `toBeTruthy` and then another test verifies that adding a file from the WordPress Media Library actually creates the desired HTML output ? Let me know. I have looked at other tests like these and I know the name of the test isn't necessarily tied to the behavior in the functionality since UI tests involve multiple components. So yes, this is somewhat a nitpick and I am fine with it being less as is, but just wanted to share these thoughts so I can gain some insights. :)

Reply:
TBH I thought about this, but didn't try it out. I ended up simply following up the Image UI Tests.
I guess we can do it if the test execution doesn't take much longer. Since these tests are so slow to execute in general, I'd say is better to write them in a way that they will execute faster.

Said that, I'll give this change a try, maybe they still execute continuously.

Reply:
I have implemented the requested changes here: https://github.com/WordPress/gutenberg/pull/27228/commits/fa3f0f13e0e52913e01ea570a4cc7da4b62399cf
It turns out the execution is continuous during the whole test suit (unexpected but nice in this particular case).

Thanks for the request! 🎉

Reply:
> It turns out the execution is continuous during the whole test suit (unexpected but nice in this particular case).

This is pretty cool, to be honest 😄 

> Thanks for the request! 🎉

Great job. Thanks @etoledom 😄